tests/baselines/reference/emitExponentiationOperatorInTemplateString1.js

```diff
@@ -0,0 +1,60 @@
+//// [emitExponentiationOperatorInTemplateString1.ts]
+var t1 = 10;
+var t2 = 10;
+var s;
+`Exp: ${t1 ** t2} abc`;
```

Comment:
Additionally, checking for exponentiation *applied to* a template expression is important as well:

* Use templates on both the left and right sides.
* Cover the above cases.
* Cover the case with a no-substitution template literal.